src/sql/azdata.proposed.d.ts

```diff
@@ -81,6 +81,10 @@ declare module 'azdata' {
 		export interface INotebookMetadata {
 			connection_name?: string;
 		}
+
+		export interface ICellMetadata {
+			connection_name?: string;
```

Comment:
So now we have a cell-level connection name and a notebook-level one? What's the scenario for when each would be used?

Reply:
Yep. For notebooks with only one connection, we'll use the notebook-level connection name and for notebooks with multiple connections, we'll use the cell-level connection name.